docs/publish/deploy-office-add-in-sso-to-azure.md

```diff
@@ -0,0 +1,265 @@
+---
+title: Deploy an Office Add-in that uses single sign-on (SSO) to Microsoft Azure App Service | Microsoft Docs
+description: Learn how to deploy an Office Add-in that uses single sign-on (SSO) to Microsoft Azure App Service from Visual Studio Code.
+ms.date: 12/20/2020
+ms.localizationpriority: medium
+---
+
+# Deploy an Office Add-in that uses single sign-on (SSO) to Microsoft Azure App Service
+
+Office Add-ins that use SSO require a web service that supports running the REST APIs and server-side code in the project. You can't deploy to a static website. Follow the steps in this article to deploy your Office Add-in to Azure App Service for staging or deployment.
+
+## Prerequisites
+
+The steps in this article work for an Office Add-in created by the [Yeoman Generator for Office Add-ins](https://github.com/OfficeDev/generator-office) using the `Office Add-in Task Pane project supporting single sign-on (localhost)` project type. Be sure you have configured the add-in project so that it runs on localhost successfully. For more information, see the [Single sign-on (SSO) quick start](../quickstarts/sso-quickstart.md).
+
+The steps in this article also require:
+
+- An Azure account. Get a trial subscription at [Microsoft Azure](https://azure.microsoft.com/free/).
+- [Azure Account extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.azure-account) for VS Code.
+- [Azure App Service extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice) for VS Code.
+
+## Create the App Service app
+
+The following steps set up a basic deployment of the Office Add-in. There are multiple ways to configure deployment that are not covered in this documentation. For additional options on how you may want to configure your deployment, see [Deployment Best Practices](/azure/app-service/deploy-best-practices)
```

Comment:
```suggestion
The following steps set up a basic deployment of the Office Add-in. There are multiple ways to configure deployment that aren't covered in this documentation. For additional options on how to configure your deployment, see [Deployment Best Practices](/azure/app-service/deploy-best-practices).
```

Reply:
@davidchesnut, looks like this suggestion was passed over. Style aside, the missing period is important.